08-Placing-Barriers/01-Barriers.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<p>Same comment as above.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/#comment-1392966255'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>The figure was supposed to be self-evident. I added many more verbose words to describe the figure.</p>